TYPES.md

```diff
@@ -9,7 +9,45 @@ There are certain types of data that Simperium uses which have specific constrai
 | **app id** | | | `a-z` `A-Z` `0-9` `.` `_` `-` | case insensitive |
 | **bucket name** | 1 | 64 | `a-z` `A-Z` `0-9` `.` `_` `-` `%` | |
 | **ccid** | 1 | 256 | `a-f` `A-F` `0-9` | [type 4 (random) UUID](https://tools.ietf.org/html/rfc4122) in hex |
-| **change type** | | | `M` `-` `DROP` `EMPTY` | see _change types_ below|
+| **change type** | | | `M` `-` `EMPTY` `DROP` | see _change types_ below|
 | **client id** | 1 | 256 | | |
 | **entity key** | 1 | 256 | `a-z` `A-Z` `0-9` `.` `_` `-` `%` `@` | |
 | **entity version** | | | integer | |
+
+## Change types
+
+There are four primary change types available to the Simperium protocol.
+Of these, `DROP` is special in that it requires administrative privileges on the bucket to be able to successfully execute.
+
+### Modify entity - `M`
```

Comment:
The preferred term was changed from `entity` to `object`. Simperium pre-websockets used the `entity` vocab so there are remnants of it and source.